R/quo.R

```diff
@@ -40,7 +40,18 @@ quo_c <- function(...) {
 #'
 #' @export
 expr_c <- function(...) {
-  inputs <- unlist(list(...), recursive = TRUE)
+  # transform single expression into list of expression
+  inputs <- map(
+    list(...),
+    function(x) {
+      if (typeof(x) != "list") {
+        list(x)
+      } else {
+        x
+      }
+    }
+  )
+  inputs <- purrr::flatten(inputs)
```

Comment:
I agree that we should replace it in the future but I think we can not do it now because we claim that admiral works with the three latest versions of R and corresponding packages.